control/lane_departure_checker/README.md

```diff
@@ -63,15 +63,18 @@ This package includes the following features:
 
 ### Node Parameters
 
-| Name                          | Type   | Description                                                   | Default value |
-| :---------------------------- | :----- | :------------------------------------------------------------ | :------------ |
-| update_rate                   | double | Frequency for publishing [Hz]                                 | 10.0          |
-| visualize_lanelet             | bool   | Flag for visualizing lanelet                                  | False         |
-| include_right_lanes           | bool   | Flag for including right lanelet in borders                   | False         |
-| include_left_lanes            | bool   | Flag for including left lanelet in borders                    | False         |
-| include_opposite_lanes        | bool   | Flag for including opposite lanelet in borders                | False         |
-| include_conflicting_lanes     | bool   | Flag for including conflicting lanelet in borders             | False         |
-| road_border_departure_checker | bool   | Flag for checking if the vehicle will departs the road border | False         |
+| Name                       | Type           | Description                                                                                        | Default value |
+| :------------------------- | :------------- | :------------------------------------------------------------------------------------------------- | :------------ |
+| will_out_of_lane_checker   | bool           | Check whether ego vehicle footprint will depart from lane                                          | True          |
```

Comment:
I think the following description is clearer.
`Enable checker whether ego vehicle footprint will depart from lane`

Reply:
changed in https://github.com/autowarefoundation/autoware.universe/pull/4861/commits/0907dfbf1d7b9fd10912a318fe5170ac88714b1b